### ISSUE TITLE
Black image result

### ISSUE THREAD

**Issue by Liampour** [ltkong218/SAFNet#5]
Hello,
congratulations for your awesome work!
When I am trying to insert my own tif files I am getting a black image as result.
Am I missing something?
Thank you for your effort, and I look forward to your reply.

**Comment by ltkong218** [comment]
Thanks for your attention!

I think there are many reasons to generate black images result:
1. The predicted .hdr file is saved in linear domain, a tone mapping method is need to visualize. 2. The provided code is based on fusion coefficients in [Kalantari 17](https://dl.acm.org/doi/10.1145/3072959.3073609). If you change the fusion coefficients, you need to train your specific SAFNet model from scratch on your data with your fusion method. 3. The provided checkpoints on [Kalantari 17](https://dl.acm.org/doi/10.1145/3072959.3073609) and Challenge123 may be overfitted on these small datasets to some extent, which can not generalize well to real data in visualization.

**Comment by Liampour** [comment]
The test conducted without changing the exposure coefficients but probably the value of zero exposure of the inputted image doesn't match to the zero value of Kalantari's dataset.
So a potential solution may be to retrain the model on larger dataset? 
I tried some tone mapping methods but the results were not like the example results. Actually the image remained black with some different shades in some regions.
Could you please suggest some tone mapping method whose results match and highlight the capabilities of this model?

**Comment by ltkong218** [comment]
Yes, I think you need to retrain the model on training dataset containing enough dynamic and lighting scenes. Exposure coefficients and HDR fusion methods can be adjust to your data.

For diverse tone mapping methods, you can refer to [Photomatix](https://www.hdrsoft.com/).
